### PR TITLE
feat: GIDK-751 | Prateek | Test | Voucher adjustments

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts [hunk 1]
```diff
@@ -1278,6 +1267,7 @@ export class NewLedgerEntryPanelComponent implements OnInit, OnDestroy, OnChange
             } else {
                 this.isAdjustReceiptSelected = false;
             }
+            this.isAdjustVoucherSelected = false
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts:1270]: add semicolon

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.ts [hunk 2]
```diff
@@ -1715,11 +1730,10 @@ export class UpdateLedgerEntryPanelComponent implements OnInit, AfterViewInit, O
                 console.log(this.vm.selectedLedger);
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.ts:1730]: remove console

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts [hunk 3]
```diff
@@ -153,7 +152,11 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
                             }
                         });
                     } else {
-                        this.toaster.warningToast("There is no advanced receipt for adjustment.");
+                        if (this.isVoucherModule) {
+                            this.toaster.warningToast("There is no advanced receipt for adjustment.");
+                        } else {
+                            this.toaster.warningToast("There is no voucher for adjustment.");
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts:158]: Please add this message in variable if used in multiple places

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts [hunk 4]
```diff
@@ -153,7 +152,11 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
                             }
                         });
                     } else {
-                        this.toaster.warningToast("There is no advanced receipt for adjustment.");
+                        if (this.isVoucherModule) {
+                            this.toaster.warningToast("There is no advanced receipt for adjustment.");
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts:156]: Please add this message in variable if used in multiple places

**Pull request merged**